back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -207,7 +208,6 @@ def update_options!(field, options_params, errors, field_index)
             next unless option
           end
           update_option_image!(option, option_params[:image_id])
-          option.move_to_bottom
```

Comment:
This was basically performing an unneeded update regardless of whether anything had changed.

Reply:
Does this part have a big impact on performance?

In general, it would be better to avoid setting the ordering manually, because this can lead to inconsistent data (I think we've had some cases of inconsistent orderings in the past which are hard to debug). But the approach looks fine, so it's acceptable if we have a significant improvement on performance.

Reply:
Yes it was the main issue for a form with many options